jvb/src/main/java/org/jitsi/videobridge/ice/Harvesters.java

```diff
@@ -83,7 +81,7 @@ public static void initializeStaticConfiguration()
 
 
             singlePortHarvesters
-                    = SinglePortUdpHarvester.createHarvesters(Config.port());
+                    = SinglePortUdpHarvester.createHarvesters(IceTransport.config.getPort());
```

Comment:
Feels weird that `Harvesters` accesses the config via `IceTransport`. Maybe move `IceTransport.config` to `IceConfig.config`?

Reply:
Sure